src/mds/Migrator.cc

```diff
@@ -1407,7 +1407,7 @@ void Migrator::export_frozen(CDir *dir, uint64_t tid)
   // send.
   it->second.state = EXPORT_PREPPING;
   mds->send_message_mds(prep, it->second.peer);
-  assert (g_conf()->mds_kill_export_at != 4);
+  ceph_assert(g_conf()->mds_kill_export_at != 4);
```

Comment:
oic. `mds_kill_export_at` is used for fault injection. so it should not be optimized out.